package.json

```diff
@@ -26,10 +27,10 @@
   },
   "homepage": "https://github.com/maxogden/menubar",
   "dependencies": {
-    "electron-positioner": "^2.0.1",
-    "extend": "^2.0.0"
+    "electron-positioner": "^2.0.3",
+    "extend": "^3.0.0"
```

Comment:
I think it's always good to have updated dependencies as far as they don't break your code. I see no point of don't updating. 😄 
Is it going to be a problem?